input/docs/cs/Porovnani_ostrosti_obrazu.md

```diff
@@ -0,0 +1,37 @@
+﻿title: Porovnání ostrosti dvou obrazů
+--- 
+# Rozmazání obrázku, hledání hran
+ ``` matlab
+A=imread('kytka256.jpg');
+% vytvoření gausovského filtru, 12 je velikost filtru, 10 je sigma neboli směrodatná odchylka
+f=fspecial('gaussian',12,10);
+A_rozmazana=imfilter(A,f); % aplikace vytvořeného filtru na obrázek
+
+A_rozm_seda=rgb2gray(A_rozmazana);
+A_seda=rgb2gray(A);
+% funkce edge hledá hrany na obrázku, vrací binární obraz
+A_rozm_E=edge(A_rozm_seda);
+A_seda_E=edge(A_seda);
+
+ ```
+# Porovnání ostrosti podle hran
+Základní předpoklad je, že u ostřejšího obrazu najde funkce edge více hran
+ ``` matlab
+ % matice vrácená funkcí edge je binární (obsahuje jen 0 a 1). 1 reprezentuje hranu. 
+ % stačí porovnat, ve které matici je více jedniček.
+C=find(A_seda_E==1); 
+B=find(A_rozm_E==1);
+if (length(C)>length(B))
```

Comment:
Když bude hran stejně, výsledek bude špatně. Neřeš to, jenom to zmiň v komentáři ("v případě stejného počtu hran je výsledek nesprávný" nebo tak něco)

Reply:
všechno by mělo být opravené - soubor je přejmenovaný, výsledek pro případ stejného počtu hran jsem upravila, výsledný obrázek z matlabu je uložený v /docs/media, u issue je odkaz na teno PR.